src/utils/helper.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+function isEmptyObject(obj) {
+  return JSON.stringify(obj) == JSON.stringify({})
```

Comment:
I decided to use the double equals, `==`, so that it allows type coercion and use it for loose quality comparison. Triple equals `===`, in this case, won't work because `JSON.stringify` returns a string.

Reply:
@davideroffo I think you're getting confused here. JSON.stringify will only ever return a string, so it is guaranteed that the right and left operands of this equality comparison are going to be strings. Hence, there is clearly no need to use the loose comparison operator, which in any case it's to be avoided and I'm surprised that eslint is not reporting that as an error.